components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -539,21 +562,49 @@ during post consent handling to get the original ConsentClaimsData object (Assum
                         " for client_id: " + clientId + " of tenantDomain: " + spTenantDomain);
             }
 
+            Map<String, Object> params;
             if (hasPromptContainsConsent(oauth2Params)) {
+                if (OAuth2Util.isDiagnosticLogsEnabled()) {
+                    params = new HashMap<>();
+                    params.put("clientId", clientId);
+                    params.put("prompt", oauth2Params.getPrompt());
+                    OAuth2Util.log(params, "SUCCESS", null, "hand-over-to-consent-service",
+                            null);
```

Comment:
can merge to 1 line, please check others as well

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1737/commits/f7b766bb97ea3607cbc320c6d7b284f05d5668ba

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1737/commits/f7b766bb97ea3607cbc320c6d7b284f05d5668ba